src/FsAutoComplete/LspServers/IFSharpLspServer.fs

```diff
@@ -19,30 +19,29 @@ type OptionallyVersionedTextDocumentPositionParams =
 [<Interface>]
 type IFSharpLspServer =
   inherit ILspServer
-  abstract FSharpSignature: TextDocumentPositionParams -> Async<LspResult<PlainNotification>>
```

Comment:
All of the changes to these PlainNotification endpoints need to be synced with Ionide.  The good news is that we can do the safety checks in Ionide ahead of this PR merging.

Reply:
(longer term we should also give these actually useful data models...)

Reply:
This PR has been opened now.

Reply:
Ionide PR merged, so this can go at our leisure. I want to update FsToolkit and pick up @TheAngryByrd's new and improved `asyncResultOption` CE to make the handlers look a bit nicer if possible, though.